src/main/flight/imu.c

```diff
@@ -201,7 +201,7 @@ static float invSqrt(float x)
 static void imuMahonyAHRSupdate(float dt, float gx, float gy, float gz,
                                 bool useAcc, float ax, float ay, float az,
                                 bool useMag,
-                                bool useCOG, float courseOverGround, const float dcmKpGain)
+                                float cogYawGain, float courseOverGround, const float dcmKpGain)
```

Comment:
IIRC the error quaternion is rotation in around UP direction in earth frame, so it should affect heading only. Definition of yaw is a bit more complicated (nonzero roll/pitch)

But the gain is more related to CoG parameter of IMU

Reply:
Good points.  

If its OK I'll keep the name the same everywhere, since it makes 'searching' for this particular parameter/concept a bit easier when looking for places in the code where we are dealing with it.